kubetest/main.go

```diff
@@ -110,6 +111,7 @@ type options struct {
 func defineFlags() *options {
 	o := options{}
 	flag.Var(&o.build, "build", "Rebuild k8s binaries, optionally forcing (release|quick|bazel) strategy")
+	flag.Var(&o.buildIngressGCE, "build-ingressgce", "Build ingress-gce binaries")
```

Comment:
Are there any options? If not then it kind of seems like this can just be a bool?

Reply:
Done.